### PR TITLE
section 2 content

### PR DESCRIPTION
**Pull request by diego-torres**

### CODE REVIEW

**Review by rsriniva — APPROVED** [review]
Approved

**Review by jramcast — COMMENTED** [review]
Good job Diego. I just left a comment regarding model frameworks. Can we clarify that part a little bit more? 

Other than that, LGTM.

File: modules/chapter1/pages/section2.adoc [hunk 1]
```diff
@@ -1,3 +1,188 @@
...
+====
+It is recommended to use a workbench that was created with the **Standard Data Science** Notebook image.
+====
+
+. Make sure that the workbench environment serves the required python packages for the notebook to run, for this to happen, open a terminal and run the following command to verify that the packages are already installed:
++
+```shell
+  pip install -r /opt/app-root/src/rhods-qc-apps/4.rhods-deploy/chapter2/requirements.txt
+```
+
```
Comment (jramcast) [inline modules/chapter1/pages/section2.adoc:61]: We might want to suggest them to execute the command in the kernel process of the notebook:
```suggestion

[TIP]
====
You might also want to execute the preceding command in the notebook kernel by using the `%pip` syntax in the notebook. 
Alternatively, you can create a custom notebook image that includes the `skl2onnx` package.
====
```

File: modules/chapter1/pages/section2.adoc [hunk 2]
```diff
@@ -1,3 +1,188 @@
...
+Now that the model is ready to use, we can make an inference using the REST API
+
+. Assign the route to an environment variable in your local machine, so that we can use it in our curl commands
++
+```shell
+export IRIS_ROUTE=https://$(oc get routes -n iris-project | grep iris-model | awk '{print $2}')
+```
+
+. Assign an authentication token to an environment variable in your local machine
```
Comment (jramcast) [inline modules/chapter1/pages/section2.adoc:173]: Is it possible to use the token provided in the RHODSAI dashboard as well? You can mention that possibility too in a note or a tip.

File: modules/chapter1/pages/section2.adoc [hunk 3]
```diff
@@ -1,3 +1,188 @@
...
++
+image::model-server-with-token.png[Model Server with token]
+
+== Deploy The Model
+
+. At the right side of the **Model Server**, we can find the **Deploy Model** button, let's click the **Deploy Model** button, to start filling the **Deploy Model** form:
++
+image::deploy-model-button.png[Deploy Model button]
+
+. Fill the **Deploy Model** from as in the example:
```
[the remark this answers is not in the document]
Comment (diego-torres) [reply modules/chapter1/pages/section2.adoc:140]: probably I can repeat the note from session 1, lines 89 to 106? https://github.com/RedHatQuickCourses/rhods-deploy/blob/main/modules/chapter1/pages/section1.adoc?plain=1#L105

**Pull request merged**